stdlib/concurrent/futures/process.pyi

```diff
@@ -1,28 +1,164 @@
+    def __init__(self, future: Future[Any], fn: Callable[..., Any], args: Iterable[Any], kwargs: Mapping[str, Any]) -> None: ...
+
+class _ResultItem(object):
+    work_id: int
+    exception: Exception
+    result: Any
+    def __init__(self, work_id: int, exception: Exception | None = ..., result: Any | None = ...) -> None: ...
+
+class _CallItem(object):
+    work_id: int
+    fn: Callable[..., Any]
+    args: Iterable[Any]
+    kwargs: Mapping[str, Any]
+    def __init__(self, work_id: int, fn: Callable[..., Any], args: Iterable[Any], kwargs: Mapping[str, Any]) -> None: ...
+
+if sys.version_info >= (3, 7):
+    class _SafeQueue(Queue[Future[Any]]):
+        pending_work_items: MutableMapping[int, _WorkItem]
+        shutdown_lock: threading.Lock
+        thread_wakeup: _ThreadWakeup
+        if sys.version_info >= (3, 9):
+            def __init__(
+                self,
+                max_size: int | None = ...,
+                *,
+                ctx: BaseContext,
+                pending_work_items: MutableMapping[int, _WorkItem],
+                shutdown_lock: threading.Lock,
+                thread_wakeup: _ThreadWakeup,
+            ) -> None: ...
+        else:
+            def __init__(
+                self, max_size: int | None = ..., *, ctx: BaseContext, pending_work_items: MutableMapping[int, _WorkItem]
+            ) -> None: ...
+        def _on_queue_feeder_error(self, e: Exception, obj: _CallItem) -> None: ...
+
+def _get_chunks(*iterables: Any, chunksize: int) -> Generator[Tuple[Any], None, None]: ...
+def _process_chunk(fn: Callable[..., Any], chunk: Tuple[Any, None, None]) -> Generator[Any, None, None]: ...
+def _sendback_result(
+    result_queue: SimpleQueue[_WorkItem], work_id: int, result: Any | None = ..., exception: Exception | None = ...
+) -> None: ...
 
 if sys.version_info >= (3, 7):
-    from ._base import BrokenExecutor
+    def _process_worker(
+        call_queue: Queue[_CallItem],
+        result_queue: SimpleQueue[_ResultItem],
+        initializer: Callable[..., None] | None,
+        initargs: Tuple[Any, ...],
+    ) -> None: ...
+
+else:
+    def _process_worker(call_queue: Queue[_CallItem], result_queue: SimpleQueue[_ResultItem]) -> None: ...
+
+if sys.version_info >= (3, 9):
+    class _ExecutorManagerThread(threading.Thread):
+        thread_wakeup: _ThreadWakeup
+        shutdown_lock: threading.Lock
+        executor_reference: weakref.ref[Any]
+        processes: MutableMapping[int, Process]
+        call_queue: Queue[_CallItem]
+        result_queue: SimpleQueue[_ResultItem]
+        work_ids_queue: Queue[int]
+        pending_work_items: MutableMapping[int, _WorkItem]
+        def __init__(self, executor: ProcessPoolExecutor) -> None: ...
+        def run(self) -> None: ...
+        def add_call_item_to_queue(self) -> None: ...
+        def wait_result_broken_or_wakeup(self) -> tuple[Any, bool, str]: ...
+        def process_result_item(self, result_item: Any) -> None: ...
```

Comment:
As far as I can see we could type this using a protocol. Until we do, we should leave this unannotated:

```suggestion
        def process_result_item(self, result_item) -> None: ...
```

Alternatively, it would make sense to define a `_ResultItem = Any` alias for now and use this here and for `wait_result_broken_or_wakeup()`.